### ISSUE TITLE
TopicPage as subtype of WebPage

### ISSUE THREAD

**Issue by tnosugar** [schemaorg/schemaorg#2572]
In news media, a very important class of pages are topical hub pages that aggregate articles on a particular topic. These need not be just simple paginated list pages, but more complex arrangements of articles with hero and feature blocs, lists, etc. In WordPress, these pages are organized around tags and cateogries.
At the moment, the CollectionPage comes closest to meeting this need, but it's not a complete fit.

**Comment by WeaverStever** [comment]
Collectively, these tags are called Folksonomy tags https://en.wikipedia.org/wiki/Folksonomy

> Collaborative tagging (also known as group tagging) is tagging performed by a group of users. This type of folksonomy is commonly used in cooperative and collaborative projects such as research, content repositories, and social bookmarking.

The tags are unique within a domain and then externalized as a collection. The ampersand is generally used as a shortcut (alias) to a username while the hashtag is generally a topic. Categories on the other hand, are generally internal collections.

Additionally, there is often a many to one relationship, i.e., there can be many hashtag topics included in one webpage and they can be included in any webpage or webpage subtype. (An author may have similar ampersand tags on different domains, so multiples could also occur.)

As such, these folksonomies would probably be better organized as a Property of CreativeWork imho.

I agree that these tags are important as many people have online personas but do not own a unique domain and the schema at the moment is domain centric.

**Comment by tnosugar** [comment]
> Collectively, these tags are called Folksonomy tags https://en.wikipedia.org/wiki/Folksonomy
> 
> > Collaborative tagging (also known as group tagging) is tagging performed by a group of users. This type of folksonomy is commonly used in cooperative and collaborative projects such as research, content repositories, and social bookmarking.
> 
> The tags are unique within a domain and then externalized as a collection. The ampersand is generally used as a shortcut (alias) to a username while the hashtag is generally a topic. Categories on the other hand, are generally internal collections.
> 
> Additionally, there is often a many to one relationship, i.e., there can be many hashtag topics included in one webpage and they can be included in any webpage or webpage subtype. (An author may have similar ampersand tags on different domains, so multiples could also occur.)
> 
> As such, these folksonomies would probably be better organized as a Property of CreativeWork imho.
> 
> I agree that these tags are important as many people have online personas but do not own a unique domain and the schema at the moment is domain centric.

Nope, I wasn't referring to folksonomies, but the conscious effort of a content owner to organize content around a single topic on a particular page.
E.g.: https://balkaninsight.com/turkey-and-the-balkans/
(But it might as well be Donald Trump (person), Porsche 911 (product) or  The United Nations (organization). 

The tag page does not represent the entity itself, but is a collection of content about that entity.

Each article on this page is tagged with "Turkey and the Balkans". They all have other tags attached to them, which will make them show up on these other pages as well.

The community is not involved in tagging the content.

Basically, every publisher, which includes any product/service website with an elaborate content strategy, would need to be able to organize content around entities of interest to them and their audience.

Marking it as a TopicPage would make more clear the function of the page, rather than the generic WebPage, or inadequate CollectionPage

**Comment by RichardWallis** [comment]
@tnosugar what you seem to be describing is a [Collection](https://schema.org/Collection) of Things (articles, images, etc.) [about](https://schema.org/about) a Thing (Donald Trump, United Nations, Porche 911).

That Collection is then the [mainEntity](https://schema.org/mainEntity) of a page that is also about the Thing in question.

Potentially the Collection could additionally be defined as, or about, a [DefinedTerm](https://schema.org/DefinedTerm) be used to index/relate things more broadly.

**Comment by tnosugar** [comment]
> @tnosugar what you seem to be describing is a [Collection](https://schema.org/Collection) of Things (articles, images, etc.) [about](https://schema.org/about) a Thing (Donald Trump, United Nations, Porche 911).
> 
> That Collection is then the [mainEntity](https://schema.org/mainEntity) of a page that is also about the Thing in question.
> 
> Potentially the Collection could additionally be defined as, or about, a [DefinedTerm](https://schema.org/DefinedTerm) be used to index/relate things more broadly.

Hm. Thanks, I'll zoom into that.

**Comment by tnosugar** [comment]
> @tnosugar what you seem to be describing is a [Collection](https://schema.org/Collection) of Things (articles, images, etc.) [about](https://schema.org/about) a Thing (Donald Trump, United Nations, Porche 911).
> 
> That Collection is then the [mainEntity](https://schema.org/mainEntity) of a page that is also about the Thing in question.
> 
> Potentially the Collection could additionally be defined as, or about, a [DefinedTerm](https://schema.org/DefinedTerm) be used to index/relate things more broadly.

So, like this? @RichardWallis 

"@type": "WebPage",
"name": "Topic: Turkey in the Balkans - BalkanInsight.com",
"mainEntity":{
          "@type": "Collection",
          "name": "Turkey in the Balkans",
 etc..

**Comment by WeaverStever** [comment]
@tnosugar

It seems like you are talking about a tag-cloud.  Wouldn't it be easier to decorate the aggregate page that the CMS (in this case WordPress) generates for the topic?

**Comment by tnosugar** [comment]
> @tnosugar
> 
> It seems like you are talking about a tag-cloud. Wouldn't it be easier to decorate the aggregate page that the CMS (in this case WordPress) generates for the topic?

Sure. I was just suggesting that a TopicPage type would make sense to publishers, as well as search engine crawlers, as topics are a building block of the knowledge graph.
